package.json

```diff
@@ -11,7 +11,6 @@
     "alphaville-auth-middleware": "Financial-Times/alphaville-auth-middleware#1.5.1",
     "alphaville-express": "Financial-Times/alphaville-express#1.1.1",
     "alphaville-header-config": "Financial-Times/alphaville-header-config#1.2.0",
-    "alphaville-marketslive-wordpress-api": "Financial-Times/alphaville-marketslive-wordpress-api#0.4.0",
```

Comment:
Does this mean we can also archive this project/repo?

Reply:
Umm yes? I think so because the only other place it's being used is in `alphaville-es-interface-service` and I'm in the process of removing it from that app.